app/models/activity.rb

```diff
@@ -0,0 +1,11 @@
+# The object which represents the user's activity. This is meant to be called by the notifiers of
```

Comment:
This is not meant to be directly accessed by client code, use the Notifications Framework.